### PR TITLE
Fix documentation for EXCEPT/INTERSECT

### PR DESCRIPTION
**Pull request by aaneja**
ALL support was added in https://github.com/trinodb/trino/pull/5890  
Documentation wasn't updated however

### CODE REVIEW

**Review by findepi — APPROVED** [review]

**Comment by aaneja** [comment]
@findepi  I signed the CLA yesterday

**Comment by colebow** [comment]
I'll make sure that gets looked at ASAP.

**Comment by martint** [comment]
@cla-bot check

File: docs/src/main/sphinx/sql/select.rst [hunk 1]
```diff
@@ -702,18 +702,17 @@ to combine the results of more than one select statement into a single result se
 
 .. code-block:: text
 
-    query INTERSECT [DISTINCT] query
+    query INTERSECT [ALL | DISTINCT] query
 
 .. code-block:: text
 
-    query EXCEPT [DISTINCT] query
```
Comment (martint) [inline docs/src/main/sphinx/sql/select.rst:709]: There's an extra space after `ALL`

Comment (aaneja) [reply docs/src/main/sphinx/sql/select.rst:709]: Fixed

**Comment by aaneja** [comment]
This is ready to merge

**Pull request merged**